Packs/Okta/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -952,6 +979,89 @@ def clear_user_sessions_command(client, args):
     return readable_output, {}, raw_response
 
 
+def get_zone_command(client, args):
+    raw_response = client.get_zone(args, args.get('zoneID', ''))
+    if not raw_response:
+        return 'No zones found.', {}, raw_response
+    zones = json.loads(raw_response)
+    readable_output = tableToMarkdown('Okta Zones', zones, headers=[
+                                      'name', 'id', 'gateways', 'status', 'system', 'lastUpdated', 'created'])
+    outputs = {
+        'Okta.Zone(val.id && val.id === obj.id)': createContext(zones)
+    }
+    return (
+        readable_output,
+        outputs,
+        zones
+    )
+
+
+def list_zones_command(client, args):
+    raw_response = client.list_zones(args)
+    if not raw_response:
+        return 'No zones found.', {}, raw_response
+    zones = json.loads(raw_response)
+    readable_output = tableToMarkdown('Okta Zones', zones, headers=[
+                                      'name', 'id', 'gateways', 'status', 'system', 'lastUpdated', 'created'])
+    outputs = {
+        'Okta.Zone(val.id && val.id === obj.id)': createContext(zones)
+    }
+    return (
+        readable_output,
+        outputs,
+        zones
+    )
+
+
+def applyZoneUpdates(args, zoneObject):
```

Comment:
pass specific args to the function instead of just `args`.
args we pass only to `command` functions
`def applyZoneUpdates(zone_name, gateway_ips , proxy_ips , zone_object):`